app/controllers/foreignincome/dividends/DividendsSectionCheckYourAnswersController.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.foreignincome.dividends
+
+import controllers.actions._
+import controllers.exceptions.SaveJourneyAnswersFailed
+import service.PropertySubmissionService
+import models.{ForeignDividends, JourneyPath, ReadForeignDividendsByCountry, JourneyContext, ForeignDividendsByCountry}
+import models.requests.DataRequest
+import pages.foreign.Country
+import pages.foreignincome.CountryReceiveDividendIncomePage
+import play.api.i18n.Lang.logger
+
+import javax.inject.Inject
+import play.api.i18n.{MessagesApi, I18nSupport}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import uk.gov.hmrc.play.language.LanguageUtils
+import viewmodels.checkAnswers.foreignincome.dividends._
+import viewmodels.govuk.all.SummaryListViewModel
+import views.html.foreignincome.dividends.DividendsSectionCheckYourAnswersView
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DividendsSectionCheckYourAnswersController @Inject()(
+                                       override val messagesApi: MessagesApi,
+                                       identify: IdentifierAction,
+                                       getData: DataRetrievalAction,
+                                       requireData: DataRequiredAction,
+                                       val controllerComponents: MessagesControllerComponents,
+                                       view: DividendsSectionCheckYourAnswersView,
+                                       languageUtils: LanguageUtils,
+                                       propertySubmissionService: PropertySubmissionService,
```

Comment:
```suggestion
                                       propertySubmissionService: PropertySubmissionService
```
Extra comma failing checks